views/index.ejs

```diff
@@ -1,52 +1,72 @@
-    <% if (approveSlotsFull) { %>
-    <p class="text-warning lead">
-      <img src="/images/sad_panda.gif" alt="Sad Panda" class="img-circle">&nbsp;I Am a sad panda. I will never find any slots for acceptance.
-    </p>
-    <% } %>    
-    <% finishedStory.forEach(function(story){ %>
-      <%- include('story/summary', {story: story, status: "finished", message: "In Review"}); %>
-      <% }); %>
-    </div>    
+</head>
+<body>
+<div class="container-fluid" role="main">
+    <!--<div class="page-header">-->
+        <!--<h1>Current Stories</h1>-->
+    <!--</div>-->
+
     <div class="row">
-    <% if (reviewSlotsFull) { %>
-    <p class="text-warning lead">
-      <img src="/images/sad_panda.gif" alt="Sad Panda" class="img-circle">&nbsp;I Am a sad panda.  I will never find any slots for review.
-    </p>
-    <% } %>    
-    <% story.forEach(function(story){ %>
-      <%- include('story/summary', {story: story, status: "started", message: "In Progress"}); %>
-      <% }); %>
+        <div class="col-md-4">
+            <h2>Play</h2>
+
+            <% if (reviewSlotsFull) { %>
+                <p class="text-warning lead">
+                    <img src="/images/sad_panda.gif" alt="Sad Panda" class="img-circle">&nbsp
+                    I Am a sad panda. I will never find any slots for review.
+                </p>
+            <% } %>
+
+            <% story.forEach(function(story) { %>
+                <%- include('story/summary', {story: story, status: "started", message: "In Progress"}); %>
+            <% }); %>
+        </div>
+
+        <div class="col-md-4">
+            <h2>Review</h2>
+
+            <% if (approveSlotsFull) { %>
+                <p class="text-warning lead">
+                    <img src="/images/sad_panda.gif" alt="Sad Panda" class="img-circle">
+                    I Am a sad panda. I will never find any slots for acceptance.
+                </p>
+            <% } %>
+
+            <% finishedStory.forEach(function(story) { %>
+                <%- include('story/summary', {story: story, status: "finished", message: "In Review"}); %>
+            <% }); %>
+        </div>
+
+        <div class="col-md-4">
+            <h2>Approval</h2>
+
+            <% deliveredStory.forEach(function(story) { %>
+                <%- include('story/summary', {story: story, status: "delivered", message: "In Approving"}); %>
+            <% }); %>
+        </div>
     </div>
 
-  
     <footer class="footer">
-        <p>
-            <a href="https://www.pivotaltracker.com/n/projects/<%= projectId %>" target="_blank">Full Fat Pivotal Buzz</a>. All the javascripts all of the time.
+        <p class="text-center">
+            <a href="https://www.pivotaltracker.com/n/projects/<%= projectId %>" target="_blank">
+                Full Fat Pivotal Buzz
+            </a>
         </p>
     </footer>
-    </div><!-- /container -->
-  </body>
+</div><!-- /container -->
+
+<!-- I am terribly sorry for this... Really sorry... - Rafal -->
```

Comment:
lol, that's fine. let's include it in source though, instead of via the CDN, add it to public/js/